teleband/courses/api/views.py

```diff
@@ -414,3 +415,36 @@ def unassign(self, request, **kwargs):
             )
 
         return Response(status=status.HTTP_200_OK)
+
```

Comment:
I know you said add this as its own thing to not have to deal with permissions, however after learning about django more I believe this makes more sense than starting new viewset.